index.md

```diff
@@ -13,6 +13,21 @@ seo:
 
 The content covers everything about machine translation, from products to research, and from history to news.
 
+<div style="display: flex; justify-content: center; gap: 20%;">
+  <div>
+    <center>
+      <h1>44</h1>
+      <a href="/apis">APIs</a>
```

Comment:
Could we perhaps link them to `apis.md` and `languages.md`?

Reply:
This effectively does.  Since it's HTML not markdown, it makes more sense to just use the final path, like we did further below.